lib/routes/content.js

```diff
@@ -187,27 +188,26 @@ const routes = application => {
      * @param {object} next - Express callback
     */
     app.get(`/${config.content.base}/:type/:id/:revision/${config.content.actions.edit}`, (req, res, next) => {
-      const errors = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].errors`, {});
-      const idSess = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].id`, {});
-      const revisionSess = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].revision`, {});
-      const values = _.get(req.session, `form.content.add[${req.params.type.toLowerCase()}].content`, {});
+      const errors = _.get(req.session, `form.content.edit[${req.params.type.toLowerCase()}].errors`, {});
+      const id = (_.get(req.session, `form.content.edit[${req.params.type.toLowerCase()}].id`) || _.get(req, 'params.id'));
```

Comment:
Two questions here, for both 192 and 193:
* Why are you using an or statement `||` instead of a default fallback for `_.get` like was previously used?
* Where are these values used? I can't seem to find another use for them, but I do see that this is duplicated in a few places (`id` specifically)